ObjectWithoutClassType is missing in mapPHPStanTypeToDocString()

```bash
 [ERROR] Could not process "/var/www/crawler-1/Classes/Hooks/IndexedSearchCrawlerHook.php" file, due to:                
         "Rector\NodeTypeResolver\StaticTypeMapper::mapPHPStanTypeToDocString for PHPStan\Type\ObjectWithoutClassType"
```

--- a/packages/NodeTypeResolver/src/StaticTypeMapper.php
+++ b/packages/NodeTypeResolver/src/StaticTypeMapper.php
@@ -323,6 +323,10 @@ final class StaticTypeMapper
             return $phpStanType->getClassName();
         }
 
+        if ($phpStanType instanceof ObjectWithoutClassType) {
+            return 'object';
+        }
+
         if ($phpStanType instanceof ClosureType) {
             return '\\' . Closure::class;
         }

